runners/google-cloud-dataflow-java/examples/build.gradle

```diff
@@ -87,6 +87,61 @@ task preCommitFnApiWorker(type: Test) {
   }
 }
 
+task postCommitFnApiWorkerJava11(type: Test) {
```

Comment:
I see that this is a similar task to `preCommitFnApiWorker`. Could we reuse `preCommitFnApiWorker` and parametrize it? My main concern now is that we duplicate lots of code that will be hard to maintain. 

When refactoring this, please remember to set default argument/parameter values to java 8's in order not to need to change anything else in the code that already uses this. Also please change its name to `fnApiWorkerTests` or something similar because it's clearly not preCommit only now.

Reply:
After looking into this task implementation, I am unable to see any difference.

So I am also wondering, why we need that additional task at all. Wouldn't it be sufficient to just call the other?

So maybe rename `preCommitFnApiWorker`to something like `runFnApiWorker` and call that task in respective Jenkins job definitions?

Reply:
You're right, that didn't add anything. I renamed `preCommitFnApiWorker` to `verifyFnApiWorker` and removed my redundant tasks. Now both precommit (already existent) and postcommit (new) jobs use the same tasks. distinction between Java 11 and Java 8 is done by setting a flag in the Jenkins job